package.json

```diff
@@ -36,8 +36,7 @@
     "test:node": "npm run build:commonjs && npm run mocha-local -- \"test-node/**/*.js\"",
     "test:browser": "npm run karma-local start karma.conf.js",
     "test:dev": "npm run karma-local start karma-dev.conf.js",
-    "test:travis": "npm run karma-local start karma-travis.conf.js",
-    "test:all": "npm run test:node && npm run test:travis",
+    "test:all": "npm run test:node && npm run test:browser",
```

Comment:
Same as above - please add back a script for GH CI such as "test:gh"

Reply:
77f5c6b